clients/lsp-terraform.el

```diff
@@ -62,6 +65,30 @@
                   :priority -1
                   :server-id 'tfls))
 
+
+;; terraform-ls
+
+(defgroup lsp-terraform-ls nil
+  "LSP support for Terraform, using terraform-ls from Hashicorp."
+  :group 'lsp-mode
+  :link '(url-link "https://github.com/hashicorp/terraform-ls")
+  :package-version `(lsp-mode . "8.0.1"))
+
+(defcustom lsp-terraform-ls-server "terraform-ls"
+  "Path to the `terraform-ls' binary."
+  :group 'lsp-terraform-ls
+  :risky t
+  :type '(choice
+          (file :tag "File")
+          (repeat string))
+  :package-version `(lsp-mode . "8.0.1"))
+
+(lsp-register-client
+ (make-lsp-client :new-connection (lsp-stdio-connection `(,lsp-terraform-ls-server "serve"))
+                  :major-modes '(terraform-mode)
+                  :priority 1
```

Comment:
do you really want to have that server with a bigger priority?

Reply:
I have no preference, I chose `1` because `-1` is already taken. I can change to `-2` if it makes any sense. In my experience both servers are so so. But the Hashicorp one's repo seems to be more active recently.

Reply:
It is really up to you to pick the server we recommend to the users.